components/style/properties/longhand/font.mako.rs

```diff
@@ -1081,6 +1084,16 @@ ${helpers.single_keyword_system("font-variant-caps",
                 }
             }
         }
+
+        impl AnimatedValueAsComputed for T {
+            #[inline]
+            fn restrict_value(self, restriction_type: Restriction) -> Self {
```

Comment:
AFAIK there is never a need for such a method given Animatable methods can clamp directly.